app/helpers/order_helper.rb

```diff
@@ -0,0 +1,13 @@
+module OrderHelper
+  def get_cart
```

Comment:
you shouldn't need to put get_ at the beginning of functions cause... obviously if a function is named cart it would return the cart